File: teamscale-client/src/test/java/com/teamscale/client/TestDataTest.java
package com.teamscale.client;

import org.junit.jupiter.api.Test;

class TestDataTest {

	@Test
	public void ensureHashingDoesNotThrowException() {
		new TestData.Builder().addByteArray(new byte[]{1, 2, 3}).addString("string").build();
	}

}

File: teamscale-client/src/test/kotlin/com/teamscale/client/TestDataTest.kt
package com.teamscale.client

import org.junit.jupiter.api.Test

internal class TestDataTest {
	@Test
	fun ensureHashingDoesNotThrowException() {
		TestData.Builder().addByteArray(byteArrayOf(1, 2, 3)).addString("string").build()
	}
}